blog/2020-04-02-march-dev-update.md

```diff
@@ -0,0 +1,46 @@
+---
+title:  "March, 2020 - Dev Update"
+author: daniel
+author_url: https://github.com/da-kami
+author_image_url: https://avatars1.githubusercontent.com/u/5557790
+tags: [dev-update]
+---
+
+> The sky is everywhere, it begins at your feet.
+>
+> — Jandy Nelson
+
+[Join us for the next COMIT Community ☕️](https://meet.jit.si/COMIT-Community-Coffee-04-2020) today, 7pm Sydney time, or listen in on [youtube](https://youtu.be/DHbGsf9i8TM)!
+
+On today's community coffee we will reflect on the first quarter of this year with a focus on Lightning Network (layer 2) integration into COMIT.
+We will discuss advantages and possible pitfalls. 
+Furthermore, we will discuss decentralised order-book solutions and what we want to tackle in the second quarter.
+
+<!--truncate-->
+
+We are moving fast.
+But sometimes things need time - the Lightning integration into the COMIT core is almost done... and will be finished shortly.
+In order to speed things up for integrating Lightning we now have a squad of three people working on that task only.
+With the focus squad you can be expected within the next week.
+Once the [Lightning integration epic](https://github.com/coblox/admin/issues/158) is closed it's done! Stay tuned.
```

Comment:
This statement is true but suggests that this ticket will be closed next week, which is not going to be the case.